codegen/type_converter_test.go

```diff
@@ -68,7 +68,15 @@ func (r *naivePackageNameResolver) TypePackageName(
 }
 
 func newTypeConverter() *codegen.TypeConverter {
-	return codegen.NewTypeConverter(&naivePackageNameResolver{})
+	// construct fake request helper
+	requestHelper := codegen.ConvertOptions{
```

Comment:
we need to pass `fromStruct` and `toStruct` into this method and then use them.

We can also make `FromSuffix` just `ConvertRequest`